src/main/io/ledstrip.c

```diff
@@ -76,42 +77,28 @@
 
 #include "telemetry/telemetry.h"
 
-PG_REGISTER_WITH_RESET_FN(ledStripConfig_t, ledStripConfig, PG_LED_STRIP_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(ledStripConfig_t, ledStripConfig, PG_LED_STRIP_CONFIG, 1);
```

Comment:
This shouldn't be needed - appending parameters to the end of a parameter group is supported without an increment of the version. See https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Thanks - done.